woocommerce/woocommerce-android
# [Wear App] Remove NavController injection from all Wear ViewModels

Summary
==========
Removes from the Wear app architectural design the practice of injecting the NavController into every VIewModel with a navigation action. This is not recommended due to the fact that the UI grants a NavController, and when the screen is destroyed, the NavController reference will also be lost. 

This PR fixes it by keeping all NavControllers inside the Composables. The result is a simpler way to create the Wear ViewModels, without the need of using assisted injections.

How to Test
==========
Make sure all screens with navigations (Login and Order List) are still working, even during configuration changes

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
